coordinator/src/test/scala/filodb.coordinator/queryplanner/MultiPartitionPlannerSpec.scala

```diff
@@ -125,200 +131,414 @@ class MultiPartitionPlannerSpec extends AnyFunSpec with Matchers with PlanValida
 
   }
 
-  it ("should generate all PromQlRemoteExec plan for TopLevelSubquery") {
+  it ("should generate simple plan for one local partition for TopLevelSubquery") {
```

Comment:
Can you please add one considerably complicated and comprehensive test case to PlannerHierarchySpec ?